e2e_example/test/build_integration_test.dart

```diff
@@ -31,4 +31,21 @@ void main() {
       expect(result.stdout, contains('Skipping tests due to build failure'));
     });
   });
+
+  group('config validation', () {
```

Comment:
nit: It might be better to move this to `build_runner/test/generate/build_integration_test.dart` because you can have a lot fewer dependencies and the test will run faster (and also not run ddc or anything else).

Reply:
Done